### PR TITLE
DCS-1093 added middleware to protect update routes from missing cookie

### PR DESCRIPTION
**Pull request by louisenorris**

### CODE REVIEW

File: backend/routes/amendBooking/index.ts [hunk 1]
```diff
@@ -31,14 +34,19 @@ export default function createRoutes({ bookingService, availabilityCheckService,
...
-  router.get('/video-link-not-available/:bookingId', asyncMiddleware(videoLinkNotAvailable.view()))
-  router.post('/video-link-not-available/:bookingId', asyncMiddleware(videoLinkNotAvailable.submit()))
+  router.get('/video-link-not-available/:bookingId', checkUpdatePresent, asyncMiddleware(videoLinkNotAvailable.view()))
+  router.post(
+    '/video-link-not-available/:bookingId',
+    checkUpdatePresent,
+    asyncMiddleware(videoLinkNotAvailable.submit())
+  )
 
   router.get('/room-no-longer-available/:bookingId', asyncMiddleware(roomNoLongerAvailable.view()))
```
Comment (andrewrlee) [inline backend/routes/amendBooking/index.ts:44]: probably want to wrap this too? maybe we aren't in create and need to?

Comment (louisenorris) [reply backend/routes/amendBooking/index.ts:44]: We're not currently wrapping it in create but can add that as well

File: backend/routes/amendBooking/videoLinkNotAvailableController.test.ts [hunk 2]
```diff
@@ -60,13 +60,6 @@ describe('video link is not available controller', () => {
...
-    it('should redirect if no state', async () => {
-      const req = mockRequest({ params: { bookingId: '123' } })
-      req.signedCookies = {}
-      await controller.view()(req, res, null)
-      expect(res.redirect).toHaveBeenCalledWith('/booking-details/123')
-    })
-
     it('should redirect back to booking search page if have since selected an available option', async () => {
       const req = mockRequest({
         params: { bookingId: '123' },
```
Comment (GurnankCheema) [inline backend/routes/amendBooking/videoLinkNotAvailableController.test.ts:65]: should bookingId be a number?

Comment (louisenorris) [reply backend/routes/amendBooking/videoLinkNotAvailableController.test.ts:65]: It's a string when it's part of the params - just checked in the debugger to be sure :)

**Review by GurnankCheema — DISMISSED** [review]
LGTM

File: backend/routes/amendBooking/videoLinkNotAvailableController.test.ts (hunk 2 above)
Comment (GurnankCheema) [reply backend/routes/amendBooking/videoLinkNotAvailableController.test.ts:65]: great

File: backend/routes/amendBooking/index.ts (hunk 1 above)
Comment (louisenorris) [reply backend/routes/amendBooking/index.ts]: Sorted in second commit

**Review by bell-pa — APPROVED** [review]
LGTM

**Pull request merged**